lib/cb/utils/validator.rb

```diff
@@ -64,6 +63,16 @@ def try_parse_xml(body)
       rescue MultiXml::ParseError
         nil
       end
+
+      def error_message(processed_response)
+        if processed_response['errors']
+          processed_response['errors']
+        elsif processed_response['Errors']
+          processed_response['Errors']
+        else
+          ''
+        end
```

Comment:
Using `.fetch` I think we could have done this with 1 line, but it really doesn't matter :)